pkg/gateway/operations/putobject.go

```diff
@@ -177,6 +177,12 @@ func handleUploadPart(w http.ResponseWriter, req *http.Request, o *PathOperation
 			}
 		}
 
+		if srcRepo.StorageID != o.Repository.StorageID {
+			o.Log(req).WithField("copy_source", copySource).WithError(err).Error("copy between different blockstores is not allowed")
```

Comment:
I don't understand "WithError(err)" - is there an error?

Also please rephrase the error message as above.